src/components/Modal/ModalContent/AddMovieForm/AddMovieForm.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+export default function AddMovieForm() {
+    return(
+        <form action="#" className="add-movie-form">
+            <h1>Add movie</h1>
+            <div className="form-row">
+                <label htmlFor="title">Title</label>
+                <input id="title" type="text" placeholder="Title" defaultValue="Moana"/>
+            </div>
+            <div className="form-row">
+                <label htmlFor="release-date">Release date</label>
+                <input id="release-date" type="date" placeholder="Select date"/>
+            </div>
+            <div className="form-row">
+                <label htmlFor="movie-url">Movie URL</label>
+                <input id="movie-url" type="text" placeholder="Movie URL here"/>
+            </div>
+            <div className="form-row">
+                <label htmlFor="genre">Genre</label>
+                <div className="select">
+                    <select id="genre" defaultValue='DEFAULT'>
```

Comment:
I think it would be a good thing to move dropdown to separate common component which can be easily re-used. (keep in mind that options shouldn't be hardcoded. it's better to have some array of options that can be passed as props to this Dropdown component and rendered within `.map()`)

Reply:
Currently, I don't reuse this component. I just added an array with select options and rendered within .map()